app/models/playlist_item.rb

```diff
@@ -1,6 +1,11 @@
 class PlaylistItem < ActiveRecord::Base
   belongs_to :stream
   validates :song, uniqueness: true, unless: :last_song_differs
+  after_create {|item| item.notify }
+
+  def notify
+    $redis.publish 'songs', { song: self }.to_json
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.